instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/__init__.py

```diff
@@ -66,6 +66,8 @@ def request_hook(span, service_name, operation_name, api_params):
     def response_hook(span, service_name, operation_name, result):
         # response hook logic
 
+    sanitize_query (bool) - an optional query sanitization flag, default is True
```

Comment:
The default is false right?
self.sanitize_query = kwargs.get("sanitize_query", False)

Reply:
Typo thanks